sqlalchemy_json/__init__.py

```diff
@@ -50,14 +50,28 @@ def coerce(cls, key, value):
             return NestedMutableList.coerce(key, value)
         return super(cls).coerce(key, value)
 
+class MutableListOrDict(Mutable):
```

Comment:
This should maintain 2 empty lines between definitions (and I guess I should really include `black` or `flake8` with a plugin in the CI runs..)